docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -4,7 +4,8 @@ description: How to configure Teleport to automatically create transient host us
 ---
 
 Teleport's SSH Service can be configured to automatically create local Unix users
-upon login.
+upon login.  Refer to the [Desktop Access RBAC guide](../desktop-access/rbac.mdx#automatic-user-creation)
+if looking for instructions on how to automatically create local users in Windows machines.
```

Comment:
I'd like @ptgott or @alexfornuto's review for the copy here. Something about _if looking for instructions_ doesn't seem quite right to me but I can't even describe why.

Reply:
I can drop the `if looking` to make it more direct, open to opinions here so style stays consistent across our docs

Reply:
reworded to `for how to automatically create local users in Windows machines`, will let docs team review then close out